pkg/internal/checkup/checkup.go

```diff
@@ -87,7 +87,7 @@ func (c *Checkup) Setup(ctx context.Context) error {
 	return nil
 }
 
-func (c *Checkup) Run(ctx context.Context) error {
+func (c *Checkup) Run(ctx context.Context, oslatLatencyThreshold time.Duration) error {
```

Comment:
The config is passed into `New()`.
Please add a config field to `Checkup` and use it here.
IMO we should not add additional params to `Run()` because it will not scale well.

Reply:
I figured we can do this later, but let's make our future selves an easier life..
DONE